### DISCUSSION TITLE
Balance display error.

### DISCUSSION THREAD

**Discussion by Poligr4f** [Chia-Network/chia-blockchain#17798] [Support]
After the transfer of funds, the balance of the wallet is zero. The "Resync" function did not help. There may be difficulties with the wallet itself. Reinstalling did not solve the problem. Also, when trying to add a token, the message "the wallet needs to be fully synced" is displayed.
![xch](https://github.com/Chia-Network/chia-blockchain/assets/165310434/5d62ecff-c727-4ef5-ac07-abfeee14b1e8)

**Comment by BrandtH22** [comment]
Hey @Poligr4f , 

As a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia). 

Per the error message it seems your wallet is not fully synced, there are a number of reasons that could be the case but that is what is leading to the 0 balance and the inability to send funds or add tokens. 

To verify the wallets sync status you can look at the wallet icon in the top right corner. If green it is synced , if yellow/orange it is syncing, and if red it is not synced and likely not connected to any peers. Clicking on that icon will show you your peer connections and the data you are downloading from them. 

Generally users will find that there is a networking or configuration issue preventing the connection to or download of data from peers. 

Feel free to jump into our discord server to go through more in depth troubleshooting otherwise it will be a good idea to look at your logs for any errors or warnings `~/.chia/mainnet/log/debug.log`

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
